index.html

```diff
@@ -185,7 +185,7 @@ <h1>About Myself</h1>
                 <li>
```

Comment:
![page-title](https://user-images.githubusercontent.com/93039359/200529155-581287af-6bb8-43e1-9b58-2e6fa891eb00.PNG)

- [ ] Your page title should be strong and good in this case your page title is just Portfolio 🤔 Kindly change it to `Anna | Portfolio` to make it good and strong 😊